status/controls.py

```diff
@@ -0,0 +1,122 @@
+"""
+    Handler related to Controls page
+"""
+from status.util import SafeHandler
+from genologics import lims
+from genologics.config import BASEURI, USERNAME, PASSWORD
+
+lims = lims.Lims(BASEURI, USERNAME, PASSWORD)
```

Comment:
Nicer to rename the var something other than the name of the module, as we should have been doing everywhere else.

Reply:
Also, is this used anywhere in this page?

Reply:
This doesn't seem to be used anywhere

Reply:
I needed to get the BASEURI from somewhere. the var is actually not used, I am removing it.